sqlstore/src/main/resources/db/migration/V8101__change_seqparam_names.sql

```diff
@@ -0,0 +1 @@
+UPDATE SequencingParameters SET name = REPLACE(name, 'x', '×');
```

Comment:
Should be OICR-specific since it fixes changes made there, and it's otherwise fine for there to be an 'x' in the name... Or those changes should be pulled into develop?